pkg/network/opencontrail/config.go

```diff
@@ -45,8 +45,9 @@ func NewConfig() *Config {
 		ApiPort:          8082,
 		DefaultProject:   "default-domain:default-project",
 		PublicNetwork:    "default-domain:default-project:Public",
-		PrivateSubnet:    "10.0.0.0/16",
-		ServiceSubnet:    "10.254.0.0/16",
+		PrivateSubnet:    "10.10.0.0/16",
+		ServiceSubnet:    "10.247.0.0/16",
```

Comment:
Ah! ok. I had tried kube-network-manager --option=x, in vain. Will change.

Thanks,
Ananth

On Fri, Jun 5, 2015 at 2:16 PM, Pedro Marques <notifications@github.com>
wrote:

> In pkg/network/opencontrail/config.go
> <https://github.com/Juniper/contrail-kubernetes/pull/10#discussion_r31852329>
> :
>
> > @@ -45,8 +45,9 @@ func NewConfig() *Config {
> >  		ApiPort:          8082,
> >  		DefaultProject:   "default-domain:default-project",
> >  		PublicNetwork:    "default-domain:default-project:Public",
> > -		PrivateSubnet:    "10.0.0.0/16",
> > -		ServiceSubnet:    "10.254.0.0/16",
> > +		PrivateSubnet:    "10.10.0.0/16",
> > +		ServiceSubnet:    "10.247.0.0/16",
>
> Please don't commit this change.
> You can override this in the command line by specifying:
> kube-network-manager -- --option=x
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Juniper/contrail-kubernetes/pull/10/files#r31852329>.
>